ipaclient/install/client.py

```diff
@@ -3132,6 +3132,12 @@ def uninstall(options):
     remove_file(paths.SSSD_MC_GROUP)
     remove_file(paths.SSSD_MC_PASSWD)
 
+    try:
+        run(["sssctl","cache-remove","-o"])
+    except Exception as e:
+        raise ScriptError("sssd-tools package not installed".format(e),
```

Comment:
You are not using e anywhere in format.

Reply:
@Tiboris Please try to instead think of what this actually does and why it does not make sense instead of trying to point out code issues 😛 It also uses general `Exception` but I am definitely pointing that out 😄